miqa/core/conversion/import_export_csvs.py

```diff
@@ -22,8 +23,13 @@ def validate_file_locations(input_dict, project):
         if key == 'file_location':
             raw_path = Path(value)
             if not raw_path.is_absolute():
-                # not an absolute file path; refer to project import csv location
-                raw_path = Path(project.import_path).parent.parent / raw_path
+                import_path = (
+                    GlobalSettings.load().import_path
+                    if project == 'global'
```

Comment:
This appears to be the only place where the magic string `global` is still used. Is this left over from something?

Reply:
Yes, nice catch. changed in [e329dcd](https://github.com/OpenImaging/miqa/pull/292/commits/e329dcd50dbc882992b6d7016f8d982222d87ffa)